src/documents/Sonification.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Sonification
+subheader:
+layout: default
+category: Methods
+---
+
+## What is Sonification?
+
+Sonification is a general term for conveying information or data through non-speech audio &mdash; compare to the much better-known term "visualization". Many of us are familiar with simple sonification as an aspect of the user interface of computers or other digital devices, such as alert sounds when receiving new email.
+
+*Data sonification* is an analogous audio technique to *data visualization* where structured data is converted into another form in order to communicate aspects of often-complex information clearly and efficiently, to support specific analysis of aspects of the data, or to show patterns and relationships in the data.
+
+## The Case for Sonification in Open Educational Resources
+
+While visualization is commonly used in many areas of education, sonification is a more theoretical and experimental area. This makes it both challenging and exciting as a potential means of supporting ["the creation of variations and enhancements"](/Techniques.html#AllowTheCreationOfVariationsAndEnhancements) in OER.
+
+Sonification's most obvious audience would seem to be the learner with visual impairments, but it can also be appreciated by those with cognitive impairments, learning disabilities, or a preference for audio presentation. It can help supplement data visualizations or open new areas for discussion, research and learning.
+
```

Comment:
Blast from the past as I try to close this loop - maybe something from http://www.ascilite.org/conferences/sydney10/procs/Sankey-full.pdf, @jhung?